frameworks/cassandra/universe/config.json

```diff
@@ -89,7 +89,7 @@
         "placement_constraint": {
           "type": "string",
           "description": "The Marathon-style placement constraint string controlling node placement",
-          "default": ""
+          "default": "hostname:MAX_PER:1"
```

Comment:
@ArtRand What effect does this change have? Limit us to one C* node per agent, so they can't by default stack?